docs/fiddles/features/recent-documents/main.js

```diff
@@ -3,19 +3,15 @@ const fs = require('fs')
 const path = require('path')
 
 function createWindow () {
-  const win = new BrowserWindow({
+  new BrowserWindow({
     width: 800,
-    height: 600,
-    webPreferences: {
-      nodeIntegration: true
-    }
+    height: 600
   })
-
-  win.loadFile('index.html')
```

Comment:
I feel like this example is still a bit clearer with the `index.html` page - and maybe add the directions to click on the app icon there?